COMPATIBILITY.md

```diff
@@ -60,6 +60,7 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [ ] `PerformancePublisher` (`performance`): [JENKINS-32650](https://issues.jenkins-ci.org/browse/JENKINS-32650)
 - [ ] `zephyr-for-jira-test-management` : [JENKINS-32801](https://issues.jenkins-ci.org/browse/JENKINS-32801)
 - [ ] `BapSshPublisher` (`publish-over-ssh`): [JENKINS-27963](https://issues.jenkins-ci.org/browse/JENKINS-27963)
+- [X] `PerfSigRecorder` and 5 more (`performance-signature-dynatrace`): supported as of 2.0
```

Comment:
According to [this](https://github.com/jenkinsci/performance-signature-dynatrace-plugin/commits/master) there is 2.0 release yet, so should it say _scheduled to be_ supported as of 2.0?